.env.development

```diff
@@ -5,22 +5,17 @@ REACT_APP_RELEASE_ID=$npm_package_version
 REACT_APP_RELEASE_NAME=$npm_package_versionName
 REACT_APP_NOSCRIPT_CONTENT=JavaScript is required.
 REACT_APP_NAMESPACE=dev
-REACT_APP_PATIENT_SERVICE_URL=https://patient.qa.clin.ferlab.bio/patient
-REACT_APP_GENE_SERVICE_URL=https://gene.qa.clin.ferlab.bio/gene
-REACT_APP_META_SERVICE_URL=https://meta.qa.clin.ferlab.bio/meta
-REACT_APP_FHIR_SERVICE_URL=https://fhir.qa.clin.ferlab.bio/fhir
-REACT_APP_RENDERER_SERVICE_URL=https://renderer.qa.clin.ferlab.bio/pdf
-REACT_APP_HPO_SERVICE_URL=https://hpo.qa.clin.ferlab.bio/hpo
+REACT_APP_PATIENT_SERVICE_URL=https://patient.qa.cqgc.hsj.rtss.qc.ca/patient
+#REACT_APP_PATIENT_SERVICE_URL=http:localhost:4000/patient
+REACT_APP_GENE_SERVICE_URL=https://gene.qa.cqgc.hsj.rtss.qc.ca/gene
+REACT_APP_META_SERVICE_URL=https://meta.qa.cqgc.hsj.rtss.qc.ca/meta
+REACT_APP_FHIR_SERVICE_URL=https://fhir.qa.cqgc.hsj.rtss.qc.ca/fhir
+REACT_APP_HPO_SERVICE_URL=https://hpo.qa.cqgc.hsj.rtss.qc.ca/hpo
 REACT_APP_DEFAULT_USERNAME=
 REACT_APP_DEFAULT_PASSWORD=
 REACT_APP_FHIR_ES_PATIENT_BUNDLE_ID=3531cb44-6eeb-4e3f-9eb6-710bff83dfc0
 REACT_APP_FHIR_ES_REQUEST_BUNDLE_ID=39d1814b-818e-4083-a9ff-c1580da38001
-REACT_APP_KEYCLOAK_CONFIG={"realm": "clin", "url": "https://auth.qa.clin.ferlab.bio/auth/", "clientId": "clin-client", "authClientId": "clin-acl"}
-
+REACT_APP_KEYCLOAK_CONFIG={"realm": "clin", "url": "https://auth.qa.cqgc.hsj.rtss.qc.ca/auth/", "clientId": "clin-client", "authClientId": "clin-acl"}
 HTTPS=false
-#REACT_APP_ARRANGER_API=https://data-api.qa.cqdg.ferlab.bio
-REACT_APP_ARRANGER_API=http://localhost:5050
-REACT_APP_ARRANGER_PROJECT_ID=clin_2021_11_30_v1
-SASS_PATH=./client/src/style/themes/clin
-REACT_APP_ZEPLIN_URL=https://notebook.qa.clin.ferlab.bio
-REACT_APP_FHIR_CONSOLE_URL=https://fhir-console.qa.clin.ferlab.bio/home
+REACT_APP_ARRANGER_API=https://data-api.qa.cqgc.hsj.rtss.qc.ca
+REACT_APP_ARRANGER_PROJECT_ID=clin_blue_2021_12_09_v1
```

Comment:
REACT_APP_ARRANGER_PROJECT_ID=clin_2021_12_16_v1

Reply:
(probably not used by front-end anymore)

Reply:
C'est pas 100% supprimer du code encore. ca sert a rien de la mettre a jour c'est vrai. mais ca n'a aucun impact pour le moment.